recipes/powerline-gitstatus_conda_recipe/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set build = 0 %}
+{% set version = "1.3.1" %}
+
+package:
+    name: powerline-gitstatus
+    version: {{version}}
+
+source:
+    url: https://github.com/jaspernbrouwer/powerline-gitstatus/archive/v{{version}}.tar.gz
+    sha256: f198a6947dcd76cc380e308e10a56d0988b1b3d4c69517bf141f4daa483fe367
+
+build:
+    noarch: python
+    number: {{build}}
+    script: {{PYTHON}} setup.py install
```

Comment:
Generally using `pip` is better than running the `setup.py` file directly:
http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip#use-pip